### PR TITLE
Fix Variables Usage + Change Components Names to Capital Case  

### LINKED ISSUE #2: Style components instead of using 100% scss
Before we use scss to style the live site, since we're using gatsby this year we want to use  as much style components as possible

### PR DESCRIPTION
**Pull request by alinastarkov**
### What does this implement/fix? Explain your changes.
Fix Variables Usage + Change Components Names to Capital Case  for style components #5 
HCK-6

### Does this fix any currently open issues? 
- Fixes #2 

### Where has this been tested?
**Platforms (Desktop, Phone, Tablet, etc...)**

**Browsers**

### Checklist
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by loreina** [comment]
Can you indicate which JIRA issues this solves/is associated with? Just do HCK-<num>

**Comment by loreina** [comment]
is there a way to shorten style variables?

File: src/styles/components/index.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
 import styled from "styled-components";
+import * as styleVariables from './variable'
+
+const landingColorPrimary = styleVariables.colorHackRed;
+const landingColorBorder = styleVariables.hackBlack5;
+const landingColorDarkBg = styleVariables.hackBlack5;
```
Comment (loreina) [inline src/styles/components/index.js]: should this be `LandingColorPrimary' etc ?

Comment (alinastarkov) [reply src/styles/components/index.js]: variable name = camel Case, component name = capital by convention i think

Comment (alinastarkov) [reply src/styles/components/index.js]: style variables can be shorten to svar ? stylevar ? etc

Comment (loreina) [reply src/styles/components/index.js]: what about just `style.` or is that used already?

Comment (alinastarkov) [reply src/styles/components/index.js]: style can be confused with styled from gatsy style components

Comment (loreina) [reply src/styles/components/index.js]: `svar` or anything short will work, I'll leave that up to you. I just think `styleVariables` is too long for a prefix

Comment (alinastarkov) [reply src/styles/components/index.js]: I feel like svar is not expressive enough and can be confusing to read too. I'll go with stylevars

Comment (alinastarkov) [reply src/styles/components/index.js]: just fixed it

**Pull request merged**